### DISCUSSION TITLE
Entities

### DISCUSSION THREAD

**Discussion by kylegrahammatzen** [Minestom/Minestom#1710] [Help] [answered]
I am curious to know if MineStom utilizes any entity saving capabilities within the Anvil Loader. From my limited understanding of the subject, it appears that there is no built-in serialization or deserialization for entities, indicating that it would be the responsibility of the developer to implement such functionality.

**Comment by kylegrahammatzen** [comment, marked as answer]
Answered in #off-topic, it's still suggested to submit discussions though.

**Reply by dev-hydrogen** [reply]
post the answer here too for the sake of other people

**Comment by emortaldev** [comment]
I've created a replacement anvil loader which supports most entities, however they are not re-saved yet unfortunately. If you find an entity that doesn't work properly, feels free to add more to the code.

https://discord.com/channels/706185253441634317/1038812288834097192/1044641005498015805
https://bin.bloom.host/ufulizirow.java
